packages/driver/.eslintignore

```diff
@@ -3,3 +3,4 @@
 **/package-lock.json
 **/tsconfig.json
 **/cypress/fixtures
+**/_test-output
```

Comment:
not related but sometimes when I run lint locally it errors on these files - these files are generated during some cypress tests and are already part of gitignore